backend/composer/models.py

```diff
@@ -269,6 +269,18 @@ def name(self):
             return f"{region_name},{layer_name}"
         else:
             return "Unnamed Entity"
+        
+    @property
+    def ontology_uri(self):
+        if self.simple_entity:
```

Comment:
question: @D-GopalKrishna can you think of a solution that is a oneline solution?
please check with @afonsobspinto, I asked him yesterday the same :-)